src/components/containers/BarChart.js

```diff
@@ -4,32 +4,60 @@ import {
   VictoryBar,
   VictoryChart,
   VictoryTheme,
-  VictoryLabel,
   VictoryLine,
+  VictoryAxis,
 } from "victory-native";
+import {
+  PRIMARY_MEDIUM,
+  PRIMARY_DARK,
+  TXT_MEDIUM_BG,
+} from "../../constants/colorCodes";
 
 const BarChart = ({ data, averageValue }) => {
-  const ValueLabel = (props) => {
-    const { datum } = props;
-    return <VictoryLabel {...props} text={datum.value} />;
+  const ValueLabel = () => null;
```

Comment:
Is this function necessary?

Reply:
Yes, it is necessary to remove the value displayed on top of the bar